src/compute-pr-actions.ts

```diff
@@ -208,6 +208,7 @@ function extendPrInfo(info: PrInfo): ExtendedPrInfo {
             requireMaintainer ? "maintainer"
             : info.popularityLevel === "Well-liked by everyone" ? "other"
             : info.popularityLevel === "Popular" ? "owner"
+            // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- is ahead of tsc which doesn't narrow this ternary -> "maintainer"

```

Comment:
```suggestion
            // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

TS *does* know that it's only `"Critical"` that is left now, but I specifically don't want to drop the last comparison in case `PopularityLevel` gets extended in the future.  (In that case I want YS to barf at this.)

Reply:
I get `"maintainer" | undefined` not `"maintainer" | never`? Tweaked the comment.

Reply:
I'm talking about the type of `info.popularityLevel`, which in the last clause is inferred as just `"Critical"`.  The `undefined` there is just because I wanted to make it safer to add a new popularity level in the future but I found a better hack for this anyway, and will use it when this gets in.